src/mdast/fromMarkdown.ts

```diff
@@ -2,6 +2,13 @@ import { u } from 'unist-builder';
 import type { GenericNode, GenericParent, GenericText, Spec, Token } from './types';
 import { withoutTrailingNewline } from './utils';
 
+const UNHIDDEN_TOKENS = new Set([
```

Comment:
These tokens are marked as `hidden` in `markdown-it-docutils` but are not hidden in myst mdast